src/container/srv_target.c

```diff
@@ -2574,8 +2574,16 @@ ds_cont_tgt_prop_update(uuid_t pool_uuid, uuid_t cont_uuid, daos_prop_t	*prop)
 	uuid_copy(arg.cpa_cont_uuid, cont_uuid);
 	uuid_copy(arg.cpa_pool_uuid, pool_uuid);
 	arg.cpa_prop = prop;
+	/*
+	 * Before initiating the rebuild scan, the iv snap fetch function
+	 * will be invoked. This action may prompt a collective call to up targets
+	 * whose containers have not yet been created. Therefore, we should skip
+	 * the up targets in this scenario. The target property will be updated
+	 * upon initiating container aggregation.
+	 */
```

Comment:
The comment said it's about a snap IV fetch, so shouldn't we fix the collective call in ds_cont_tgt_snapshots_update()? A property update is also invoked on rebuild scan handler?

Reply:
you are right, i think i miss-read the code when writting it.